tests/screenObjects/HomeScreen.js

```diff
@@ -13,8 +13,7 @@ const SELECTORS = {
     ? '-android uiautomator: new UiSelector().description("Home-screen")'
     // accessibility:
     : '~Home-screen',
-  TEXT: driver.isAndroid
-    ? '-android uiautomator: new UiSelector().text("{TEXT}")'
+  TEXT: driver.isAndroid? '-android uiautomator: new UiSelector().text("{TEXT}")'
```

Comment:
Missing line jump here

TEXT: driver.isAndroid 
? ""
: ""

Reply:
also. When you use the uiautomator, i think you can avoid putting new UiSelector(). I think you can use directly ".text("{TEXT}")".  Try and see if it works so the selector looks cleaner